README.md

```diff
@@ -1,11 +1,13 @@
 Simple Data Access Control
 =====================
 
-Simple Data Access Control allows the restriction of which user can access which resources, in the way compatible with Advanced Search.
+Simple Data Access Control allows the restriction of which user can access which resources, in the way compatible with
+Advanced Search.
 
 Access Privileges are granted either to users directly or to roles, applying to all users who have that specific role.
 
-Privileges are given per resource, so that in order to remove the write access to all items within a class, the new access rights need to be applied recursively to all resources by checking "recursive" before saving the changes.
+Privileges are given per resource, so that in order to remove the write access to all items within a class, the new
+access rights need to be applied recursively to all resources by checking "recursive" before saving the changes.
```

Comment:
We should explain the 2 "save strategies" we have as part of `taoDacSimple/PermissionsService.conf.php` 
```
<?php
/**
 * Default config header created during install
 */

return new oat\taoDacSimple\model\PermissionsServiceFactory(array(
    'save_strategy' => 'oat\\taoDacSimple\\model\\SyncPermissionsStrategy',
    'recursive_by_default' => false
));


```
- `SyncPermissionsStrategy` (Default) 
When saving -> overwrite what permissions existed with new ones as given from user.
- `SavePermissionsStrategy`
When saving -> merge existing permission with changes (permission addition,removals) from the user

It's worth to emphasize that "recursive" save, especially with `SyncPermissionsStrategy` is very dangerous.

Reply:
Ok, added to the doc. Thanks for the feedback